{{cookiecutter.project_slug}}/_/ci-services/definitions/.gitlab-ci.yml

```diff
@@ -49,14 +49,15 @@ stages:
   before_script:
   - export DJANGO_SECRET_KEY=$(cat /dev/urandom | tr -dc A-Za-z0-9 | head -c50)
   - export POSTGRES_PASSWORD=$(cat /dev/urandom | tr -dc A-Za-z0-9 | head -c16)
+  - export POSTGRESQL_USERNAME={{ cookiecutter.project_slug.replace('-', '_') }}
```

Comment:
It seems to me, we should align the variable names here to reduce the distraction.

Reply:
Yes, good point, that would be cleaner.